python/lsst/cp/pipe/utils.py

```diff
@@ -296,7 +296,7 @@ def makeDataRefList(self, namespace):
             self.refList += refList
 
 
-def irlsFit(initialParams, dataX, dataY, function, weightsY=None):
+def irlsFit(initialParams, dataX, dataY, function, weightsY=None, weightType='Cauchy'):
     """Iteratively reweighted least squares fit.
 
     This uses the `lsst.cp.pipe.utils.fitLeastSq`, but applies
```

Comment:
Do you need to update the docstring a bit? (e.g., "applies Cauchy by default, but there are more options, see below" or something like that).